src/utils/googleAnalytics.ts

```diff
@@ -20,8 +20,12 @@ export const loadGoogleAnalytics = (): void => {
   if (!trackingID) {
     console.error('[GoogleAnalytics] - In order to use google analytics you need to add an trackingID')
   } else {
-    GoogleAnalytics.initialize(trackingID)
-    GoogleAnalytics.set({
+    ReactGA.initialize(trackingID, {
+      gaOptions: {
+        siteSpeedSampleRate: 100,
```

Comment:
Should we remove this parameter? I think this is only for testing purposes

Reply:
yes, thx!